### DISCUSSION TITLE
Consumer don't know about ongoing data transfer by provider [provider-push]

### DISCUSSION THREAD

**Discussion by reisman234** [eclipse-edc/Connector#2446] [General]
I had a discussion with @ronjaquensel around  8 month ago about a simpe prototype related to the 04.0-file-transfer sample and how it is possible from the consumer-side not to switch into the `completed` state during an ongoing transfer of the provider. 
Back then, I got introduced to the StatusChecker which solved for me the problem at that time.

Now a now Milestone is released and I wanted to ask if there are some improvements to this feature related to all data-planes (s3, http, etc... ). At the moment i'm on ms7 and after a caill against  `transferprocess/{{TRANSFER_ID}}/state` endpoint on the consumer-side,  I see still only the state of the consumer. 

IMHO needing to know the current state of a transfer is a crucial feature in a data exchange framework and writing a StatusChecker for each dataplane is a daunting task.

**Comment by jimmarino** [comment]
For the status of features, please see the documentation and change logs. 

Improving the status checker is something we can look into after the work on the new IDS protocol is in place.
